client/stylesheets/_toCat.scss

```diff
@@ -3272,6 +3272,19 @@ On click the images will be displayed at normal size to complete the effect
     height: 500px !important;
 }
 
+.itemAfterBanner {
+    background-color: $color-purple;
+    padding: 10px;
+    width: 100%;
+    border-radius: 4px;
+    margin-bottom: 25px;
+}
+
+.itemAfterBanner b {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces